example_packages/noarch_python/meta.yaml

```diff
@@ -0,0 +1,46 @@
+package:
+  name: bokeh
+  version: 0.7.1
+
+source:
+  fn: bokeh-0.7.1.tar.gz
+  url: https://pypi.python.org/packages/source/b/bokeh/bokeh-0.7.1.tar.gz
+  md5: 426f2b0850018fab1407f2e7ed129544
+
+build:
+  noarch_python: True
+  number: 1
```

Comment:
The build number lets you create a new build of a package without incrementing the version. conda compares versions of a package first by looking at the version string and then, if those are equal, comparing build numbers.